android/build.gradle

```diff
@@ -57,25 +57,21 @@ android {
 detekt {
     version = detekt_version
     input = files("src/main/kotlin")
-    filters = ".*/resources/.*,.*/build/.*"
     dependencies {
         detektPlugins "io.gitlab.arturbosch.detekt:detekt-formatting:$detekt_version"
     }
 }
 
 protobuf {
     protoc {
-        artifact = 'com.google.protobuf:protoc:3.6.1'
-    }
-    plugins {
```

Comment:
this was really badly documented in protobuf but we shouldn't use the plugin anymore but use javalite itself